### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by AmbientLighter**

### CODE REVIEW

File: libcloud/dns/types.py [hunk 1]
```diff
@@ -69,11 +69,9 @@ class RecordType(object):
     TXT = 'TXT'
     PTR = 'PTR'
     SOA = 'SOA'
-    HINFO = 'HINFO'
     SPF = 'SPF'
     SRV = 'SRV'
     SSHFP = 'SSHFP'
-    PTR = 'PTR'
```
Comment (jimbobhickville) [inline libcloud/dns/types.py:76]: Why was this removed?  I'm pretty sure the Rackspace RDNS stuff I added recently references it.

**Comment by jimbobhickville** [comment]
All the typo and formatting fixes look good, just the question about the `RecordType` removals.

File: libcloud/dns/types.py (hunk 1 above)
Comment (AmbientLighter) [inline libcloud/dns/types.py:76]: This line is duplicated twice: `PTR = 'PTR'`
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L70
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L76

This line is duplicated twice: `HINFO = 'HINFO'`
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L68
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L72

Comment (jimbobhickville) [inline libcloud/dns/types.py:76]: Ah, derp, I missed that.  Good catch.

**Comment by jimbobhickville** [comment]
lgtm

**Comment by Kami** [comment]
Good catch - thanks.

File: docs/compute/drivers/auroracompute.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 AuroraCompute Computer Driver Documentation
-======================================
+===========================================
```
Comment (Kami) [inline docs/compute/drivers/auroracompute.rst:2]: We need to update our build process to turn sphinx warnings into errors again and fail the build.

It used to be turned on, but then for some reason (don't recall why anymore), we disabled it.

**Comment by Kami** [comment]
Merged, thanks.
